From: Michael Schutte <michi@uiae.at>
Subject: [PATCH] fixes/kbd_mode

Change misleading kbd_mode message for XLATE mode, closes: #546712.

The text said “default (ASCII) mode”, which is wrong in two respects:
XLATE isn’t limited to 7-bit ASCII, but works with various 8-bit
charsets, and the XLATE mode hasn’t been the kernel default for a while.
Thanks for the hint go to Samuel Thibault <sthibault@debian.org>.

The usage of kbd_mode together with grep in shell scripts might become
an issue with this commit, though cursory research with
codesearch.google.com shows only the BiCon project
<http://sourceforge.net/projects/bicon/>, which does “kbd_mode | grep -q
-i Unicode” and is thus unaffected.

Signed-off-by: Michael Schutte <michi@uiae.at>

---
 src/kbd_mode.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/kbd_mode.c b/src/kbd_mode.c
index b286678..fe4e233 100644
--- a/src/kbd_mode.c
+++ b/src/kbd_mode.c
@@ -89,7 +89,7 @@ main(int argc, char *argv[]){
 		printf(_("The keyboard is in mediumraw (keycode) mode\n"));
 		break;
 	      case K_XLATE:
-		printf(_("The keyboard is in the default (ASCII) mode\n"));
+		printf(_("The keyboard is in xlate (8-bit) mode\n"));
 		break;
 	      case K_UNICODE:
 		printf(_("The keyboard is in Unicode (UTF-8) mode\n"));
-- 
tg: (cfffc72..) fixes/kbd_mode (depends on: upstream/master)
